src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/ExceptionHandlingUtil.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.InvalidCaseDataException;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.callback.CallbackException;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.callback.ProcessResult;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+
+final class ExceptionHandlingUtil {
+    private ExceptionHandlingUtil() {
+        //
+    }
+
+    static CallbackException handleGenericException(Exception exception, String msg) {
+        // log happens individually to cover update/ccd cases
+        return new CallbackException(msg, exception);
+    }
```

Comment:
In my eyes this basically replaces a one liner:
```
throw new CallbackException(msg, exception);
```
with another one liner:
```
throw ExceptionHandlingUtil.handleGenericException(exception, msg)
```
while also hiding the exception type.

What's the benefit of introducing this util?

Reply:
Fixing quality gate failure due to code duplication